app/static/i18n/fr.json

```
@@ -152,7 +152,7 @@
         "HTTP_ENDPOINT": "Point de connection HTTP",
         "CPU_USAGE": "Utilisation du CPU",
         "HEAP_USAGE": "Utilisation du tas",
-        "DISK_USAGE": "Utilisation du disque",
+        "DISK_USAGE": "Utilisation de disque",
```

Actually on reflection I do think 'du' is better here
```suggestion
        "DISK_USAGE": "Utilisation du disque",
```